testing/test_config.py

```diff
@@ -348,23 +352,35 @@ def pytest_configure(config):
                 [pytest]
                 required_plugins = myplugin>1.0,<2.0
                 """,
+                "1.5",
                 None,
                 id="1-ok-pin-loose",
             ),
             pytest.param(
                 """
                 [pytest]
-                required_plugins = pyplugin==1.6
+                required_plugins = myplugin
+                """,
+                "1.5a1",
+                None,
+                id="1-ok-prerelease",
+            ),
+            pytest.param(
+                """
+                [pytest]
+                required_plugins = myplugin==1.6
                 """,
-                "Missing required plugins: pyplugin==1.6",
+                "1.5",
+                "Missing required plugins: myplugin==1.6",
                 id="missing-version",
             ),
             pytest.param(
                 """
                 [pytest]
-                required_plugins = pyplugin==1.6 other==1.0
+                required_plugins = myplugin==1.6 other==1.0
```

Comment:
The `pyplugin` name was added by @nicoddemus in #7470, but I believe this to be a typo - the test only mocks a `myplugin` plugin, and those tests sound like they intend to test different versions, not plugins which are missing entirely (there's already `other` for that!)

Reply:
You are probably right, thanks for catching it. 👍